### PR TITLE
V0.18.0 release notes

### PR DESCRIPTION
**Pull request by antalszava**
The release notes and the version bump for the v0.18.0 release.

### CODE REVIEW

**Comment by antalszava** [comment]
[ch8950]

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -346,13 +379,13 @@
   selected or deselected during local execution of tests.
   [(#1633)](https://github.com/PennyLaneAI/pennylane/pull/1633)
 
-* Hamiltonians are now natively supported on the `default.qubit` device if `shots=None`. 
+* Hamiltonians are now natively supported on the `default.qubit` device if `shots=None`.
   This makes VQE workflows a lot faster in some cases.
   [(#1551)](https://github.com/PennyLaneAI/pennylane/pull/1551)
   [(#1596)](https://github.com/PennyLaneAI/pennylane/pull/1596)
```
Comment (josh146) [inline .github/CHANGELOG.md:385]: Hmm, this one I would probably move to the 'Faster Hamiltonian' section of 'New Features'. I wonder if @soranjh has any benchmark plots we could include in order to show-off the speed improvement

File: .github/CHANGELOG.md [hunk 2]
```diff
@@ -424,10 +457,10 @@ and requirements-ci.txt (unpinned). This latter would be used by the CI.
```
Comment (josh146) [inline .github/CHANGELOG.md:457]: I can't comment, but I would remove the 'separate requirements file for the CI issue' one, since that was a change to the repo continuous integration, rather than the core PL package itself.

File: .github/CHANGELOG.md [hunk 3]
```diff
@@ -424,10 +457,10 @@ and requirements-ci.txt (unpinned). This latter would be used by the CI.
 
 * The QFT operation is moved to template
   [(#1548)](https://github.com/PennyLaneAI/pennylane/pull/1548)
```
Comment (josh146) [inline .github/CHANGELOG.md:459]:
```suggestion
* The `QFT` operation has been moved, and is now accessible via `pennylane.templates.QFT`.
  [(#1548)](https://github.com/PennyLaneAI/pennylane/pull/1548)
```

@antalszava, having a read back of it now,

- I would remove the entries specifying 'this thing has been moved' if it results in no change to the API (e.g., `qml.Hamiltonian` still has the same public import location, even though the file has changed. They are not technically improvements, and I don't think they are useful to have here, since they are not user facing.

- I would keep entries that result in a breaking change to import location, but move them to 'breaking changes'. I.e., I believe the QFT change is breaking, right?

File: .github/CHANGELOG.md [hunk 4]
```diff
@@ -424,10 +457,10 @@ and requirements-ci.txt (unpinned). This latter would be used by the CI.
...
-  
-* The `qml.ResetError` is now supported for `default.mixed` device. 
+
+* The `qml.ResetError` is now supported for `default.mixed` device.
   [(#1541)](https://github.com/PennyLaneAI/pennylane/pull/1541)
-  
+
 * `QNode.diff_method` will now reflect which method was selected from `diff_method="best"`.
   [(#1568)](https://github.com/PennyLaneAI/pennylane/pull/1568)
```
Comment (josh146) [inline .github/CHANGELOG.md:466]: I can't comment, but there are a couple of cases below where 'user' is left in the changelog entry. I would re-write them to remove usage of 'user'.

E.g.,

> An error is now raised when no arguments are passed to an observable, to inform that wires have not been supplied. (#1547)

> QNodes now include validation to warn ~users~ if a supplied keyword argument is not one of the recognized arguments. (#1496)

File: .github/CHANGELOG.md [hunk 5]
```diff
@@ -457,13 +490,13 @@ and requirements-ci.txt (unpinned). This latter would be used by the CI.
   Temporary backward compatibility has been added to support the use of `_stepsize` for one
```
Comment (josh146) [inline .github/CHANGELOG.md:490]: I can't comment, but above, `numpy` should be `NumPy`

File: .github/CHANGELOG.md [hunk 6]
```diff
@@ -457,13 +490,13 @@ and requirements-ci.txt (unpinned). This latter would be used by the CI.
...
   release cycle. `update_stepsize` method is deprecated.
   [(#1625)](https://github.com/PennyLaneAI/pennylane/pull/1625)
-  
+
 
 <h3>Bug fixes</h3>
 
 * Fix bug with norm and Jax tracers (jit) when using `QubiStateVector`.
   [(#1649)](https://github.com/PennyLaneAI/pennylane/pull/1649)
```
Comment (josh146) [inline .github/CHANGELOG.md:498]: This is not descriptive enough - suggest rewriting it to inform a reader how this impacts their workflow:
```suggestion
* Fixed a bug with where `@jax.jit` would fail on a QNode that used `qml.QubiStateVector`.
  [(#1649)](https://github.com/PennyLaneAI/pennylane/pull/1649)
```

File: .github/CHANGELOG.md [hunk 7]
```diff
@@ -506,12 +539,14 @@ and requirements-ci.txt (unpinned). This latter would be used by the CI.
...
-Vishnu Ajith, Akash Narayanan B, Thomas Bromley, Olivia Di Matteo, Sahaj Dhamija, Tanya Garg, Josh Izaac,
-Prateek Jain, Ankit Khandelwal, Christina Lee, Ian McLean, Johannes Jakob Meyer, Romain Moyard, Esteban Payares,
-Pratul Saini, Maria Schuld, Arshpreet Singh, Ingrid Strandberg, Slimane Thabet, Antal Száva, David Wierichs,
-Vincent Wong.
+Vishnu Ajith, Akash Narayanan B, Thomas Bromley, Olivia Di Matteo, Sahaj
+Dhamija, Tanya Garg, Anthony Hayes, Theodor Isacsson, Josh Izaac, Prateek Jain,
+Ankit Khandelwal, Nathan Killoran, Christina Lee, Ian McLean, Johannes Jakob
+Meyer, Romain Moyard, Lee James O'Riordan, Esteban Payares, Pratul Saini, Maria
+Schuld, Arshpreet Singh, Ingrid Strandberg, Antal Száva, Slimane Thabet, David
+Wierichs, Vincent Wong.
```
Comment (josh146) [inline .github/CHANGELOG.md:547]: A massive contributor list 💯

**Review by josh146 — CHANGES_REQUESTED** [review]
Thanks for this Antal! Looking pretty good -- I've left some suggestions regarding organization 🙂

File: .github/CHANGELOG.md (hunk 1 above)
Comment (soranjh) [reply .github/CHANGELOG.md:385]: Yes, I have some data. Please let me prepare a clean figure.

File: .github/CHANGELOG.md [hunk 8]
```diff
@@ -1,19 +1,44 @@
-# Release 0.18.0-dev (development release)
+# Release 0.18.0 (current release)
 
 <h3>New features since last release</h3>
 
+<h4>New PyTorch native device</h4>
+
+* A new pytorch device, `qml.device('default.qubit.torch', wires=wires)`, supports
+  backpropogation with the torch interface.
+  [(#1225)](https://github.com/PennyLaneAI/pennylane/pull/1360)
```
Comment (antalszava) [inline .github/CHANGELOG.md:9]:
```suggestion
  [(#1360)](https://github.com/PennyLaneAI/pennylane/pull/1360)
```

File: .github/CHANGELOG.md (hunk 1 above)
Comment (josh146) [reply .github/CHANGELOG.md:385]: Looks awesome @soranjh 💪 

Only suggestion is minor - might be good to modify the title to say

> Time per 1 VQE step for H2 on default.qubit

to avoid readers thinking this speedup occurs for hardware devices/finite-shots

Comment (soranjh) [reply .github/CHANGELOG.md:385]: image
Another version

Comment (soranjh) [reply .github/CHANGELOG.md:385]: image

Comment (josh146) [reply .github/CHANGELOG.md:385]: 😍

File: .github/CHANGELOG.md [hunk 9]
```diff
@@ -209,19 +279,9 @@
...
+  For usage details please consider the [docstring of the optimizer](https://pennylane.readthedocs.io/en/stable/code/api/pennylane.RotosolveOptimizer.html?highlight=rotosolveoptimizer#pennylane.RotosolveOptimizer).
 
-* Vector-Jacobian product transforms have been added to the `qml.gradients` package.
-  [(#1494)](https://github.com/PennyLaneAI/pennylane/pull/1494)
-
-  The new transforms include:
-
-  - `qml.gradients.vjp`
-  - `qml.gradients.batch_vjp`
+<h4>Faster and improved Hamiltonian computations</h4>
```
[the remark this answers is not in the document]
Comment (antalszava) [reply .github/CHANGELOG.md:295]: Sounds better to me too. :+1:

File: .github/CHANGELOG.md [hunk 10]
```diff
@@ -209,19 +279,9 @@
...
 
 * The Hamiltonian can now store grouping information, which can be accessed by a device to
```
[the remark this answers is not in the document]
Comment (antalszava) [reply .github/CHANGELOG.md]: Sounds good! Done

File: .github/CHANGELOG.md [hunk 11]
```diff
@@ -265,46 +325,19 @@
...
-
-      result = execute(
-          [tape1, tape2], dev,
-          gradient_fn=qml.gradients.param_shift,
-          interface="autograd"
-      )
-      return result[0] + result[1][0, 0]
+* Added a new `SISWAP` operation and a `SQISW` alias with support to the
+  `default_qubit` device.
+  [#1563](https://github.com/PennyLaneAI/pennylane/pull/1563)
```
[the remark this answers is not in the document]
Comment (antalszava) [reply .github/CHANGELOG.md]: Sounds good, done!

File: .github/CHANGELOG.md [hunk 12]
```diff
@@ -265,46 +325,19 @@
...
-  ```python
-  from pennylane.interfaces.batch import execute
-
-  def cost_fn(x):
-      with qml.tape.JacobianTape() as tape1:
-          qml.RX(x[0], wires=[0])
-          qml.RY(x[1], wires=[1])
-          qml.CNOT(wires=[0, 1])
-          qml.var(qml.PauliZ(0) @ qml.PauliX(1))
+<h4>New operation</h4>
```
[the remark this answers is not in the document]
Comment (antalszava) [reply .github/CHANGELOG.md:381]: Agreed. It doesn't really fit any of the other categories, so I've added it into Improvements. An alternative to that would be creating a general catch-all category that we can use for each release: e.g., `"Further new features"` or so.

File: .github/CHANGELOG.md [hunk 13]
```diff
@@ -265,46 +325,19 @@
...
 
 <h3>Improvements</h3>
 
+* The `frobenius_inner_product` function has been moved to the `qml.math`
+  module, and is now differentiable using all autodiff frameworks.
+  [(#1388)](https://github.com/PennyLaneAI/pennylane/pull/1388)
+
 * A warning is raised to inform the user that specifying a list of shots is
   only supported for `QubitDevice` based devices.
   [(#1659)](https://github.com/PennyLaneAI/pennylane/pull/1659)
```
[the remark this answers is not in the document]
Comment (antalszava) [reply .github/CHANGELOG.md:393]: Made the snippet smaller and in #1671  uploaded the image for the new example (https://github.com/PennyLaneAI/pennylane/pull/1671/commits/177de21e4582f547607e38610216af66b91e0d4e).

File: .github/CHANGELOG.md (hunk 1 above)
Comment (antalszava) [reply .github/CHANGELOG.md:426]: Added the latest plot in https://github.com/PennyLaneAI/pennylane/pull/1671/commits/fa6df8f21f462544bef7a8cbf12903f93b84e5b3 as part of https://github.com/PennyLaneAI/pennylane/pull/1671.

File: .github/CHANGELOG.md (hunk 2 above)
Comment (antalszava) [reply .github/CHANGELOG.md]: Done.

File: .github/CHANGELOG.md (hunk 3 above)
Comment (antalszava) [reply .github/CHANGELOG.md]: Made the changes for QFT :+1: 

For the first point, I haven't found any specific examples in the file. Missing something or just something worth keeping in mind going forward?

File: .github/CHANGELOG.md (hunk 4 above)
Comment (antalszava) [reply .github/CHANGELOG.md:498]: Done.

**Comment by antalszava** [comment]
@josh146 thank you for the suggestions here! :slightly_smiling_face: Iterated on them. Some plots are being included in https://github.com/PennyLaneAI/pennylane/pull/1671, such that the `latest` version of the docs` will be available in the release notes.

File: .github/CHANGELOG.md [hunk 14]
```diff
@@ -343,8 +336,58 @@
   res = qml.grad(cost_fn)(params)
   ```
 
+  <h4>PennyLane now comes packaged with <code>lightning.qubit</code></h4>
```
Comment (josh146) [inline .github/CHANGELOG.md:339]: @antalszava since this is a very big feature, I would add this either above or below the pytorch entry (maybe below? --- your call :) )

File: .github/CHANGELOG.md [hunk 15]
```diff
@@ -343,8 +336,58 @@
...
+   [[-1.14472273e-04  3.85963953e-02 -9.39190132e-18]
+    [-5.76791765e-05 -9.78478343e-02  0.00000000e+00]
+    [ 0.00000000e+00  0.00000000e+00  0.00000000e+00]]]
+  ```
+
 <h3>Improvements</h3>
 
+* Added a new `SISWAP` operation with support to the
+  `default_qubit` device.
+  [#1563](https://github.com/PennyLaneAI/pennylane/pull/1563)
```
Comment (josh146) [inline .github/CHANGELOG.md:385]:
```suggestion
* A new operation `qml.SISWAP` has been added, the square-root of the `qml.ISWAP` operation.
  [(#1563)](https://github.com/PennyLaneAI/pennylane/pull/1563)
```

File: .github/CHANGELOG.md [hunk 16]
```diff
@@ -357,42 +400,32 @@
...
-  <img src="https://pennylane.readthedocs.io/en/latest/_static/drawer/example_basic.png" width=70%/>
+  <img src="https://pennylane.readthedocs.io/en/latest/_static/drawer/example_release_notes.png" width=70%/>
 
 * The slowest tests, more than 1.5 seconds, now have the pytest mark `slow`, and can be
   selected or deselected during local execution of tests.
   [(#1633)](https://github.com/PennyLaneAI/pennylane/pull/1633)
 
-* Hamiltonians are now natively supported on the `default.qubit` device if `shots=None`. 
+* Hamiltonians are now natively supported on the `default.qubit` device if `shots=None`.
   This makes VQE workflows a lot faster in some cases.
```
Comment (josh146) [inline .github/CHANGELOG.md:423]: Hey @antalszava, it looks like this still needs to be moved to the "Faster, trainable, Hamiltonian simulations" section, and I can't see the plot? 🤔

File: .github/CHANGELOG.md [hunk 17]
```diff
@@ -357,42 +400,32 @@
...
   [(#1551)](https://github.com/PennyLaneAI/pennylane/pull/1551)
   [(#1596)](https://github.com/PennyLaneAI/pennylane/pull/1596)
 
-* A gradient recipe for Hamiltonian coefficients has been added. This makes it possible 
-  to compute parameter-shift gradients of these coefficients on devices that natively 
+* A gradient recipe for Hamiltonian coefficients has been added. This makes it possible
+  to compute parameter-shift gradients of these coefficients on devices that natively
   support Hamiltonians.
   [(#1551)](https://github.com/PennyLaneAI/pennylane/pull/1551)
```
Comment (josh146) [inline .github/CHANGELOG.md:430]: I think this should be deleted from here, and merged with the
```
Hamiltonians are now trainable with respect to their coefficients. (#1483)
```
entry

File: .github/CHANGELOG.md [hunk 18]
```diff
@@ -497,16 +526,16 @@ and requirements-ci.txt (unpinned). This latter would be used by the CI.
   Temporary backward compatibility has been added to support the use of `_stepsize` for one
   release cycle. `update_stepsize` method is deprecated.
   [(#1625)](https://github.com/PennyLaneAI/pennylane/pull/1625)
-  
+
 
 <h3>Bug fixes</h3>
 
 * Fix bug with shot vectors and `Device` base class.
```
Comment (josh146) [inline .github/CHANGELOG.md:533]: Very _very_ minor, but it might be nice to standardize the 'fix bug' vs 'fixed a bug' :)

File: .github/CHANGELOG.md (hunk 7 above)
Comment (josh146) [reply .github/CHANGELOG.md:586]: I just noticed that Jay is missing from this list, I recall him doing code review for bugs?

**Review by josh146 — APPROVED** [review]
Nice work Antal, happy for this to be merged in once my final comments are taken into account!

File: .github/CHANGELOG.md (hunk 14 above)
Comment (antalszava) [reply .github/CHANGELOG.md]: Moved it above, thanks!

File: .github/CHANGELOG.md (hunk 16 above)
Comment (antalszava) [reply .github/CHANGELOG.md]: Good catch, thanks! Updated this

File: .github/CHANGELOG.md (hunk 17 above)
Comment (antalszava) [reply .github/CHANGELOG.md]: Sounds good, updated.
